src/components/StaticTable/StaticTable.tsx

```diff
@@ -162,49 +165,83 @@ export const StaticTable = ({
   return (
     <div className={clsx(className, classes.root)}>
       <table className={classes.table}>
-        <thead>
-          <tr
-            className={clsx(classes.headerTr, {
-              [classes.emptyHeader]: !data.length,
-            })}
-          >
-            {columns.map(({ label, wide, narrow, noSort, leftAlign }, idx) => (
-              <th
-                key={idx}
-                className={clsx(classes.headerTh, {
-                  [classes.sortable]: sortable && !noSort,
-                  [classes.wide]: wide,
-                  [classes.narrow]: narrow,
-                  [classes.leftAlign]: leftAlign,
-                })}
-                onClick={
-                  sortable && !noSort ? () => onClickSort(idx) : undefined
-                }
-              >
-                {label}
-                {order.field === idx && sortable
-                  ? order.ascending > 0
-                    ? ' ↑'
-                    : ' ↓'
-                  : ''}
-              </th>
-            ))}
-          </tr>
-        </thead>
+        {!singleColumn && (
+          <thead>
+            <tr
+              className={clsx(classes.headerTr, {
+                [classes.emptyHeader]: !data.length,
+              })}
+            >
+              {columns.map(
+                (
+                  {
+                    label,
+                    tooltip,
```

Comment:
I wonder if, rather than heavily modifying StaticTable this way, it makes more sense to just stop using it and write your own table in AdminPage that's specific to that page's needs. When I see this ad-hoc "column data format" with tooltip fields, it makes me think we're trying to create too much abstraction/reusability, & the code would be much simpler to understand if we didn't.

Reply:
I think it'd really be better to deprecate StaticTable entirely rather than augmenting it here. It's pretty coupled with Material-UI, which we're moving away from, and honestly I find the separation of `columns` and `data` to be unnecessarily complex. If you can make your changes to AdminPage with a normal table or grid setup, we'll be pretty close to being able to remove StaticTable from the codebase entirely.

For mobile: In my experience, sometimes you want a table that changes its layout for mobile, and sometimes you just want a totally different thing to be rendered. I think it's ok to be a little repetitive, for the sake of readability & avoiding excess abstraction.

Reply:
Anyway, I appreciate you being willing to make bold changes to the code.